utils/src/main/scala/za/co/absa/hermes/utils/FileReader.scala

```diff
@@ -16,6 +16,7 @@
 package za.co.absa.hermes.utils
 
 import scala.io.Source
+import scala.language.reflectiveCalls
```

Comment:
This is interesting, I haven't seen it before.

Reply:
Stolen from ARMImplicits in Atum. Otherwise, it underlines the `.close()` method at the bottom.